ports/pcapplusplus/portfile.cmake

```
@@ -1,29 +1,30 @@
-if (VCPKG_TARGET_IS_WINDOWS)
-    vcpkg_check_linkage(ONLY_STATIC_LIBRARY)
-endif()
+# At the moment PcapPlusPlus doesn't support dynamic linkage on Windows
+set(VCPKG_LIBRARY_LINKAGE static)
+
+# Convert PcapPlusPlus to add leading zero 23.9 => 23.09
+string(REGEX REPLACE "^([0-9]+)[.]([0-9])\$" "\\1.0\\2" PCAPPLUSPLUS_VERSION "${VERSION}")
 
 vcpkg_from_github(
     OUT_SOURCE_PATH SOURCE_PATH
     REPO seladb/PcapPlusPlus
-    REF v22.11
-    SHA512 41a507ce385d8549186eeec1a1ae138070ab2021d9ffd907829551b3b865ecb526fa05a0ff9ca01b41a2a2807a60a3cba016f62063d30d849282c83e17a2b6e1
+    REF "v${PCAPPLUSPLUS_VERSION}"
+    SHA512 e7dc1dbd85c9f0d2f9c5d3e436456c2cd183fb508c869fa8fb83f46aac99b868a16283204e5d57a0bfd7587f6ac2582b3e14c6098683fad4501708c8fededd6a
     HEAD_REF master
 )
-file(COPY "${CURRENT_PORT_DIR}/CMakeLists.txt" DESTINATION "${SOURCE_PATH}")
 
 vcpkg_cmake_configure(
     SOURCE_PATH "${SOURCE_PATH}"
+    OPTIONS
+        -DPCAPPP_BUILD_EXAMPLES=OFF
 )
 
 vcpkg_cmake_install()
+vcpkg_cmake_config_fixup(PACKAGE_NAME PcapPlusPlus CONFIG_PATH lib/cmake/pcapplusplus)
```

```suggestion
vcpkg_cmake_config_fixup(CONFIG_PATH lib/cmake/pcapplusplus)
```

PACKAGE_NAME is a misnomer: It is used for the target *directory* name. We prefer lower-case dir names,  and the default is `${PORT}`.